mkdocs/docs/getting-started/your-first-experiment.md

```diff
@@ -139,29 +91,17 @@ View a report of the experiment in HTML or text formats as follows.
           -------                             |-----
           built-in/http-error-count           |0.00
           built-in/http-error-rate            |0.00
-          built-in/http-latency-max (msec)    |186.56
-          built-in/http-latency-mean (msec)   |11.25
-          built-in/http-latency-min (msec)    |4.05
-          built-in/http-latency-p50 (msec)    |6.59
-          built-in/http-latency-p75 (msec)    |7.79
-          built-in/http-latency-p90 (msec)    |9.23
-          built-in/http-latency-p95 (msec)    |10.00
-          built-in/http-latency-p99 (msec)    |183.94
-          built-in/http-latency-p99.9 (msec)  |186.30
-          built-in/http-latency-stddev (msec) |28.01
-          built-in/http-request-count         |200.00
+          built-in/http-latency-max (msec)    |203.78
+          built-in/http-latency-mean (msec)   |17.00
+          built-in/http-latency-min (msec)    |4.20
+          built-in/http-latency-p50 (msec)    |10.67
+          built-in/http-latency-p75 (msec)    |12.33
+          built-in/http-latency-p90 (msec)    |14.00
+          built-in/http-latency-p95 (msec)    |15.67
+          built-in/http-latency-p99 (msec)    |202.84
+          built-in/http-latency-p99.9 (msec)  |203.69
+          built-in/http-latency-stddev (msec) |37.94
+          built-in/http-request-count         |100.00
         ```
 
 Congratulations! :tada: You completed your first Iter8 experiment.
-
-***
-
-???+ tip "Useful variations of this experiment"
```

Comment:
I think something like this is valuable on the first experiment page. Maybe "Next Steps" or "Try Next".  It gives direction to the user about where to go. It does not have to be as complete as the current list.

Reply:
Could we hold off on this for a bit? `Your second experiment` might address many of these concerns.

Reply:
Done.